opm/simulators/wells/WellState.hpp

```diff
@@ -230,7 +233,7 @@ namespace Opm
             data::Wells dw;
             for( const auto& itr : this->wellMap_ ) {
                 const auto well_index = itr.second[ 0 ];
-                if (!this->open_for_output_[well_index])
+                if (this->status_[well_index] != Well::Status::OPEN)
```

Comment:
I am not sure this is correct. For wells that are STOPPED you would also want output I think (at least bhp makes sense even if rates are 0).

Reply:
Well - that is probably right; but the current PR should not change anything; so then this problem has always been there.

Reply:
I do not think it is a bug now, only that it may become a bug once we start actively using the STOPPED status as a possibility for this flag.

Reply:
Ok - then we fully agree, better differentiation between stop and shut is the Next pr.